kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/SchemasController.java

```diff
@@ -118,23 +117,23 @@ protected KafkaCluster getCluster(String clusterName) {
                                                                     @Valid String search,
                                                                     ServerWebExchange serverWebExchange) {
     return schemaRegistryService
-            .getAllSubjectNames(getCluster(clusterName))
-            .flatMap(subjects -> {
-              int pageSize = perPage != null && perPage > 0 ? perPage : DEFAULT_PAGE_SIZE;
-              int subjectToSkip = ((pageNum != null && pageNum > 0 ? pageNum : 1) - 1) * pageSize;
-              List<String> filteredSubjects = Arrays.stream(subjects)
-                      .filter(subj -> search == null || StringUtils.containsIgnoreCase(subj, search))
-                      .sorted()
-                      .collect(Collectors.toList());
-              var totalPages = (filteredSubjects.size() / pageSize)
-                      + (filteredSubjects.size() % pageSize == 0 ? 0 : 1);
-              List<String> subjectsToRender = filteredSubjects.stream()
-                      .skip(subjectToSkip)
-                      .limit(pageSize)
-                      .collect(Collectors.toList());
-              return schemaRegistryService.getAllLatestVersionSchemas(getCluster(clusterName), subjectsToRender)
-                      .map(a -> new SchemaSubjectsResponseDTO().pageCount(totalPages).schemas(a));
-            }).map(ResponseEntity::ok);
+        .getAllSubjectNames(getCluster(clusterName))
```

Comment:
we need this reformatting?

Reply:
Yea, next method call starts one symbol after the previous statement, not a symbol after `return`, that's weird. Also no clue why checkstyle hasn't failed before.